### PR TITLE
Add pyyaml to the requirements

### PR DESCRIPTION
**Pull request by nfraprado**
#956 moved stylesheets to yaml, but didn't add `pyyaml` as a requirement. Add it.

This is done on top of #969, since my environment is on python 3.9 and I need it to test the change, and also because when generating the `requirements.txt` with `pip-compile` to add `pyyaml`, it updates other packages much like in #969, so doing it on top of #969 keeps the diff small.

### CODE REVIEW

**Comment by lornajane** [comment]
Looks good but depends on #969 so blocked until that merges

**Comment by nfraprado** [comment]
Rebased now that #969 merged and since `requirements.txt` changes were conflicting.

**Review by lornajane — APPROVED** [review]
good catch, thanks!

**Pull request merged**
